src/Faulkner/history.py

```diff
@@ -25,6 +26,11 @@ def append(self, x):
     def get(self):
        return self.data[self.cur:]+self.data[:self.cur]
 
+    def getLastN(self, n):
+        if n <= self.cur:
+            return self.data[self.cur-n:self.cur]
```

Comment:
I'm not sure what this "data" list is suppose to hold. If "data" holds a common data structure, let's create that data structure in a different file and share it between our modules.